eng/Version.Details.xml

```diff
@@ -120,10 +120,6 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>195bf987887ea645a747ff0586440f7920c47b01</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.ILLink.Analyzers" Version="8.0.100-1.23067.1">
-      <Uri>https://github.com/dotnet/linker</Uri>
-      <Sha>c790896f128957acd2999208f44f09ae1e826c8c</Sha>
-    </Dependency>
```

Comment:
Should the corresponding version number be removed from `Versions.props`?  Otherwise I believe it will still be there and available for use, but won't ever get updated.